modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -320,6 +320,9 @@
     /** Separator for formatted coordinator properties. */
     public static final String COORDINATOR_PROPERTIES_SEPARATOR = ",";
 
+    /** Default long operations dump timeout. */
+    public static final long DFLT_LONG_OPERATIONS_DUMP_TIMEOUT = 60_000L;
```

Comment:
Currently used only in LockTimeoutObject , but should be used in IgniteKernal too:
final long longOpDumpTimeout =
            IgniteSystemProperties.getLong(IgniteSystemProperties.IGNITE_LONG_OPERATIONS_DUMP_TIMEOUT, 60_000);